src/test/java/org/springframework/data/elasticsearch/repository/query/ElasticsearchStringQueryUnitTests.java

```diff
@@ -83,6 +86,73 @@ public void shouldReplaceRepeatedParametersCorrectly() throws Exception {
 				.isEqualTo("name:(zero, eleven, one, two, three, four, five, six, seven, eight, nine, ten, eleven, zero, one)");
 	}
 
+	@Test
+	public void shouldReplaceParametersSpEL() throws Exception {
+
+		org.springframework.data.elasticsearch.core.query.Query query = createQuery("findByNameSpEL", "Luke");
+
+		assertThat(query).isInstanceOf(StringQuery.class);
+		assertThat(((StringQuery) query).getSource())
+				.isEqualTo("{ \"bool\" : { \"must\" : { \"term\" : { \"name\" : \"Luke\" } } } }");
```

Comment:
Like I wrote further down at the definition of the method, I'd prefer to have this formatted. We should then probably use `JSONAssert` to verify the equality in these tests.